### PR TITLE
azimuth: init at 1.0.2

### PR DESCRIPTION
**Pull request by marius851000**
###### Motivation for this change
Wanted to play this game, so I packaged it. 

###### Things done
I have played the game some hour without problem.
I have also quickly check the other binary.
The 3 tools have been renamed with "azimuth-" before their name to prevent conflict

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by marius851000** [comment]
Can someone help me to understand this error message ?

**Comment by samueldr** [comment]
Hmm, that steps works here manually, let's re-eval see if master was in a bad state transiently...

@GrahamcOfBorg eval

File: pkgs/games/azimuth/default.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  enableParallelBuilding = true;
+
+  # the game doesn't have an installation procedure
+  installPhase = ''
+    mkdir -p $out/bin
+    cp out/release/host/bin/azimuth $out/bin/azimuth
+    cp out/release/host/bin/editor $out/bin/azimuth-editor
+    cp out/release/host/bin/muse $out/bin/azimuth-muse
+    cp out/release/host/bin/zfxr $out/bin/azimuth-zfxr
```
Comment (matthiasbeyer) [inline pkgs/games/azimuth/default.nix]: You can simplify this by doing:

```
cp out/release/host/bin/* $out/bin/
```

File: pkgs/games/azimuth/default.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      your ship, explore the inside of the planet, fight enemies, overcome
+      obstacles, and uncover the storyline piece by piece. Azimuth features a
+      huge game world to explore, lots of little puzzles to solve, dozens of
+      weapons and upgrades to find and use, and a wide variety of enemies and
+      bosses to tangle with.
+    '';
+
+    license = stdenv.lib.licenses.gpl3Plus;
+    homepage = https://mdsteele.games/azimuth/index.html;
+    maintainers = with stdenv.maintainers; [ marius851000 ];
```
Comment (matthiasbeyer) [inline pkgs/games/azimuth/default.nix]: Use `with stdenv.lib.maintainers;` here.

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

File: pkgs/games/azimuth/default.nix (hunk 2 above)
Comment (jtojnar) [inline pkgs/games/azimuth/default.nix]: `Add platforms`.

**Review by jtojnar — CHANGES_REQUESTED** [review]

File: pkgs/games/azimuth/default.nix (hunk 1 above)
Comment (aanderse) [reply pkgs/games/azimuth/default.nix]: @matthiasbeyer if you look closely you'll notice the binaries are being renamed.

File: pkgs/games/azimuth/default.nix [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, SDL }:
+
+stdenv.mkDerivation rec {
+  name = "azimuth";
```
Comment (aanderse) [inline pkgs/games/azimuth/default.nix]: `pname = "azimuth";`

**Review by aanderse — CHANGES_REQUESTED** [review]
You also might want to consider filling an issue upstream asking for an install target.

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -2900,6 +2900,11 @@
     github = "martingms";
     name = "Martin Gammelsæter";
   };
+  marius851000 = {
```
Comment (aanderse) [inline maintainers/maintainer-list.nix]: You should rebase on master as another PR of yours has already been merged :tada:, so this is no longer required.

**Review by aanderse — CHANGES_REQUESTED** [review]

**Comment by marius851000** [comment]
@aanderse It seem to be ready

**Comment by aanderse** [comment]
@marius851000 thanks for pinging me on this
@GrahamcOfBorg build azimuth

**Review by aanderse — APPROVED** [review]

**Comment by aanderse** [comment]
Thanks @marius851000! :tada:

**Pull request merged**